tests/e2e/specs/in-game/countering.spec.js

```diff
@@ -241,8 +241,8 @@ describe('Countering One-Offs', () => {
     cy.log('Player counters again (3rd counter');
     cy.get('#counter-dialog')
       .should('be.visible')
-      .should('contain', 'Your opponent has played 2♣️ to Counter your 2♥️.')
-      .get('[data-cy=counter]')
+      .contains('Your opponent has played 2♣️ to Counter your 2♥️', {includeShadowDom: true});
```

Comment:
Might be better to abstract this in to utility if it's going to be common and/or look in to other methods of asserting here.

Reply:
I'm not sure what's the right way of doing that but the best I can come up with is to define a custom child cypress command that we can chain off of `cy.get()` to make contains assertions with `includeShadowDom` set to `true`. I'm not sure that's actually more clear though since it basically trades a one-liner for another one-liner

Reply:
I think it's best to hold off on this and keep an eye out for patterns in when we need `includeShadowDom`. Gonna hold off on making an issue pending a clearer strategy for dealing with this